public/index.html

```diff
@@ -38,7 +38,7 @@ <h4>Error</h4>
         </div>
     </nav>
     <div id="loginCard" class="row">
-        <div class="col s12 z-depth-6 card-panel">
+      <div class="col">
```

Comment:
Creo que se veía mejor con las demás clases
```suggestion
      <div class="col s12 z-depth-6 card-panel">
```